Gemfile

```diff
@@ -21,6 +21,8 @@ group :acceptance_testing do
   gem "beaker", *location_for(ENV['BEAKER_VERSION'] || '~> 2.0')
 end
 
+gem "minitest"
```

Comment:
Should this be under a test or development group so it's not required for normal usage?